### ISSUE TITLE
entgo.io/contrib - Unexpected package creation during export data loading

### ISSUE THREAD

**Issue by water-a** [ent/ent#2878]
Generating protobufs from the entproto plugin fails.

- [x] The issue is present in the latest release.
- [x] I have searched the [issues](https://github.com/ent/ent/issues) of this repository and believe that this is not a duplicate.

## Current Behavior 😯
```
go % go generate ./...
2022/08/20 20:07:20 Unexpected package creation during export data loading
--entgrpc_out: protoc-gen-entgrpc: Plugin failed with status code 1.
ent/proto/entpb/generate.go:2: running "protoc": exit status 1
```

## Expected Behavior 🤔
```
go % go generate ./...
```

## Solution

I was able to fix this issue by manually modifying the entgo.io/contrib go.mod file to upgrade golang.org/x/tools package and re-installing the proto plugin locally.

This is related to https://github.com/ent/ent/issues/2826

Tl;dr upgrading golang.org/x/tools in entgo.io/contrib fixes this issue

**Comment by rotemtam** [comment]
Hey @water-a 

Thanks for the detailed report and solution.

@all-contributors please add @water-a for bug
